dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/BundleUtil.java

```diff
@@ -51,15 +54,15 @@ public static String getStringFromBundle(String key, List<String> arguments) {
      * If it is default bundle or default metadata block #{@link DefaultMetadataBlocks#METADATA_BLOCK_NAMES}
      * method tries to get the name from default bundles otherwise it returns empty string.
      */
-    public static String getStringFromPropertyFile(String bundleKey, String bundleName) throws MissingResourceException {
+    public static String getStringFromPropertyFile(String bundleKey, String bundleName, Locale locale) throws MissingResourceException {
```

Comment:
I don't think we should change this method declaration just to be able to manipulate locales in **tests**. 
IMO it's better to add another method with additional argument which allows locale manipulation. Use this new method in tests and keep `getStringFromPropertyFile` as it is.